crates/loader/tests/ui/invalid-url-in-allowed-http-hosts.err

```diff
@@ -0,0 +1,7 @@
+Failed to load Spin app from "<test-dir>/invalid-url-in-allowed-http-hosts.toml"
+
+Caused by:
+    0: Failed to load component `hello`
+    1: One or more allowed_http_hosts entries was invalid:
+       ftp://random-data-api.fermyon.app isn't a valid host or host:port string
+       example.com/wib/wob contains a path, should be host and optional port only
```

Comment:
Does this fail because of the lack of scheme? Perhaps we could make the error message clearer?

Reply:
:shrug: this is unchanged by this PR; I just added the test

Reply:
@rylev The first one fails because there _is_ a scheme (and it's not a HTTP scheme).  The second fails because it contains a _path,_ not just a host.  Maybe they look too run-together at the moment which confuses matters?  But as Lann says this error message already exists, I can noodle on it separately.